### ISSUE TITLE
Cryptomancer-settings on other systems

### ISSUE THREAD

**Issue by fabiocosta0305** [MozillaFoundation/mpa-cryptomancer-challenge#21]
I'm a Fate RPG veteran and loved the Cryptomancer setting... But I was thinking on put the concept into other systems. What do you think?

**Comment by cryptomancer-actual** [comment]
@fabiocosta0305 Yes! Although that would be outside the purview of the Mozilla sprint, I definitely support that idea as a way to bring the game's teaching concepts to people accustomed to Fate's rules.

**Comment by fabiocosta0305** [comment]
Working on it... I'm putting this on Masters of Umdaar setting as a placeholder, but the idea is that the rules should be generic  enough to be used into any setting

**Comment by fabiocosta0305** [comment]
As soon I have an adaptation that could encompass the basics on the main elements of the setting (shards, shardnets, cryptomancy, risk eaters, the cryptomancy magic used outside the shardnets, golems) I'll publish it. I think at least two weeks will be needed to get through ot.

**Comment by fabiocosta0305** [comment]
Published my first version [in my github repo, here](https://github.com/fabiocosta0305/mpa_cryptomancer_fate). Comments are welcome. This is the first version, so it's a little crude and it hadn't undergone playtest.

**Comment by cryptomancer-actual** [comment]
Hi @fabiocosta0305 This looks solid and captures most of the important stuff related to networking and cryptography. Is there any specific feedback or help you are looking for? I don't know anything about how the Fate game rules work, but would be happy to assist in other ways.

**Comment by fabiocosta0305** [comment]
I think any suggestion, either on crunch or Fluffy will be good. I think next step will be introducing Risk Eaters for Masters of Umdaar (a Dune/Princess of Mars/Masters of Universe setting for Fate) and some other elements

**Comment by fabiocosta0305** [comment]
Did a little adventure skeleton for another setting for Fate, called [_Do: Fate of the Flying Temple_](http://www.evilhat.com/home/do-fate-of-the-flying-temple/). It's a family friendly setting, so maybe this could be a gateway for putting some privacy or cryptography basics for kids. I need to take some things a little different from Cryptomancer, but I think this could be useful somewhat for Crryptomancer

<https://github.com/fabiocosta0305/mpa_cryptomancer_fate/blob/master/the_hidden_messenger.md>

**Comment by fabiocosta0305** [comment]
If there's still interest, I'll close this. I've cloned this repo on my github. I'll put all the files there and then I'll pull request everything I've done if it's okay
